jmclient/jmclient/configure.py

```diff
@@ -73,7 +75,7 @@ def __getitem__(self, key):
 global_singleton.core_alert = core_alert
 global_singleton.joinmarket_alert = joinmarket_alert
 global_singleton.debug_silence = debug_silence
-global_singleton.config = SafeConfigParser()
+global_singleton.config = ConfigParser(strict=False)
```

Comment:
I take it you're using strict=False to match behaviour of earlier versions? I don't see any reason we'd violate strict=True, but ... I guess I could just check.

Reply:
seems to be due to populating from the defaultconfig first and then overriding, I'm not entirely sure though